### ISSUE TITLE
Printer software suite illogical initial install (Bugzilla Bug 175)

### ISSUE THREAD

**Issue by tpgxyz** [OpenMandrivaAssociation/test2#175]
This issue was created automatically with bugzilla2github
# Bugzilla Bug 175
Date: 2013-09-26 12:23:34 +0000
From: @itchka
To:   OpenMandriva QA &lt;<bugs@openmandriva.org>&gt;
CC:   @cris-b

Last updated: 2013-10-14 08:22:57 +0000

## Comment 1135
Date: 2013-09-26 12:23:34 +0000
From: @itchka

The way that printers are set up can give the impression that things are broken when in fact they are not.

The current iso after install to harddisk starts up with the hplip icon on the plasma bar. This would indicate to the user that he/she may have a printer already detected and installed or at least be able to install one. This is not the case. Trying to install a printer allow one to go through all the steps but the printer simply isn't created. I tried this several times and it kust does not work.
I then tried to install a printer in cups via the web interface this was successful however when printing a page the data appeared to go to the printer but the printer just sat there receiving data forever and not printing.

Finally I decided to try MCC. The Printer queue setup installed a further six or seven packages and suddenly everything worked including the hplip printer setup.

I do not think that HPlip should be installed by default but if it is it has additional dependencies that must also be installed.

## Comment 1136
Date: 2013-09-26 13:03:30 +0000
From: @itchka

Addendum: The HP device manager still did not show the printer when run it gave a an error message saying no printer installed.

## Comment 1153
Date: 2013-09-26 23:34:57 +0000
From: @cris-b

would be good to know exactly what packages were installed when you ran mcc.

the iso includes task-printing which pulls in all the printing packages, the only suggests in there being hplip-gui, which is now hard-coded into the list of included packages.

so basically all printing packages should be on the iso!

## Comment 1155
Date: 2013-09-27 00:50:05 +0000
From: @cris-b

confirmed.

## Comment 1156
Date: 2013-09-27 00:50:39 +0000
From: @cris-b

looks like we need system-config-printer to be installed for things to work.

added to the iso lists.

## Comment 1270
Date: 2013-10-14 08:22:57 +0000
From: @cris-b

system-config-printer is on the install/live cd by default
